Tensile/Source/lib/include/Tensile/DataTypes_Int8.hpp

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright 2019-2020 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#pragma once
+
+#ifdef TENSILE_USE_HIP
+#include <hip/hip_runtime.h>
+#endif
+
+#include <Tensile/DistinctType.hpp>
+
+namespace Tensile
+{
+    /**
+ * \ingroup DataTypes
+ */
+    struct Int8 : public DistinctType<int8_t, Int8>
+    {
+    };
+} // namespace Tensile
+
+namespace std
+{
+    inline ostream& operator<<(ostream& stream, const Tensile::Int8 val)
+    {
+        return stream << static_cast<int32_t>(val);
+    }
```

Comment:
The only purpose of adding this Int8 type is ostream operator<<.
Since when calling logTensorTyped(), we need to print the actual number as int32_t instead of int8_t (it will be printed as characters)